go/chat/types/types.go

```diff
@@ -107,7 +107,7 @@ func (j ConvLoaderJob) HigherPriorityThan(j2 ConvLoaderJob) bool {
 }
 
 func (j ConvLoaderJob) String() string {
-	return fmt.Sprintf("[convID: %s pagination: %s]", j.ConvID, j.Pagination)
+	return fmt.Sprintf("[convID: %s pagination: %v]", j.ConvID, j.Pagination)
```

Comment:
i got a `go vet` error here..
```
go vet...................................................................Failed
hookid: go-vet

go/chat/types/types.go:110: arg j.Pagination for printf verb %s of wrong type: *github.com/keybase/client/go/protocol/chat1.Pagination
exit status 1
```

Reply:
Hmm I think you should change this back, are you merged up against master? `chat1.Pagination` has a `String` function on it.